sklearn/ensemble/bagging.py

```diff
@@ -773,8 +773,8 @@ def decision_function(self, X):
 
         if self.n_features_ != X.shape[1]:
             raise ValueError("Number of features of the model must "
-                             "match the input. Model n_features is {1} and "
-                             "input n_features is {2} "
+                             "match the input. Model n_features is {0} and "
+                             "input n_features is {1} "
```

Comment:
this means it was not covered by any test. Can you update existing test to catch this pb?

thx